assignments/callbacks.js

```diff
@@ -24,26 +26,30 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.getLength)
```

Comment:
i think maybe you meant on line 31 to `return cb(arr.length)` ... not (arr.getLength) :) .. arr.length will return the length of array, arr.getLength is chaining the getLength function to the end of the array being passed in, but isn't invoked or calling anthing either..  I think i know what you meant to put, i hope this made sense :)